Create April Fools' day article (#4951)
ppy/osu-wiki: wiki/History_of_osu!/April_Fools/en.md

---

## 2019

### Sneezing girl sound effect (2019)

For April Fools' day 2019 there was approximately a 1 in 20 chance of hearing the sound effect of a high-pitched girl sneezing upon opening a beatmap [\[69,70,71\]](#references).

---

## 2020

### MillhioreF joins the Featured Artists

[MillhioreF](https://osu.ppy.sh/users/941094)—a long-time osu! moderator, developer and [Easy mod](/wiki/Game_modifier/Easy) player—was announced in [a newspost](https://osu.ppy.sh/home/news/2020-04-01-new-featured-artist-millhioref) to have "joined" the list of [Featured Artists](/wiki/Featured_Artists) as "Millhiore Firianno Biscotti" on 1 April 2020 with a selection of five songs to boot [\[72\]](#references):

- Waltz o' the Irish
- The Waltzing Irishman
- An Irish Waltz
- A Waltz From The Geographical Region Known as Ireland but Also as Éire
- There's Gold Beneath Your Waltzing Rainbow (feat. Mismagius)

> ["MillhioreF - Waltz o' the Irish (MillhioreF)"](https://osu.ppy.sh/beatmapsets/73348#osu/326585)—a [long-running joke beatmap](#2009) within the community—was also Loved on 31 March 2020 as part of this joke.

Reviewer: I don't see any particular relation between this map and the one for the April Fools' 2009, except for both being mapped by MillhioreF. The way you worded it gives the impression that Lemon Tree and Waltz o' the Irish are the same beatmap.
```suggestion
["MillhioreF - Waltz o' the Irish"](https://osu.ppy.sh/beatmapsets/73348#osu/326585) was also [Loved](/wiki/Beatmap/Category#loved) on 31 March 2020 as part of this joke.
```

Author: this was just me being stupid; I misread the title

![](img/2012-hidden.gif "What the website would have looked like with the Hidden mod enabled")

### Bad Apple ranking chart

The "Bad Apple Ranking Chart" was a joke ranking chart based on a variety of beatmaps that featured the song "Bad Apple!!" as part of osu!'s 2012 April Fools' joke. Announced via [a newspost](https://osu.ppy.sh/community/forums/posts/1431905) on 1 April 2012, the ranking chart actually functioned as a real working chart at the time and displayed the top 40 players who obtained the highest [ranked score](/wiki/Score#ranked-score) in any one of the handpicked selections of "Bad Apple!!" beatmaps [[20,21,22]][r]. An excerpt from the aforementioned newspost is inscribed below:

> We decided to dedicate this chart to the greatest song and video ever created, Bad Apple!!. You can find the awesome chart here
>
> Now, as this is a chart of epic proportions, we needed to make our prize a little better this time! The winners will receive a Lily White poster and a hand drawn picture of Reimu. (A printer is required to receive these prizes)
>
> Look forward to our next charts, the Renai Circulation and Irony charts!

— Cyclone, "Bad Apple Ranking Chart!" [[21,22]][r]

> The chart was opened on 1 April 2012 and closed on 2 April 2012. At the end of the ranking period, player [Mesita](https://osu.ppy.sh/users/201459) was the number one player with a ranked score of 145,623,328 [[23]][r].

Reviewer 2: ```suggestion
The chart was opened on 1 April 2012 and closed on 2 April 2012. At the end of the ranking period, [Mesita](https://osu.ppy.sh/users/201459) was the number one player with a ranked score of 145,623,328 [[23]][r].
```

The beatmaps included in the ranking chart are listed below:

- [nomico - Bad Apple!! (James)](https://osu.ppy.sh/beatmapsets/6252)
- [REDALiCE - Bad Apple!! (Rena-chan)](https://osu.ppy.sh/beatmapsets/10353)
- [Masayoshi Minoshima ft. nomico - Bad Apple!! (Ephemeral)](https://osu.ppy.sh/beatmapsets/10435)
- [Masayoshi Minoshima ft. nomico - Bad Apple!! (ignorethis)](https://osu.ppy.sh/beatmapsets/13177)
- [Masayoshi Minoshima feat. StrawbellyCake - Bad Apple!! (German Version) (Larto)](https://osu.ppy.sh/beatmapsets/13664)
- [Masayoshi Minoshima feat. Larto & nomico - Awesome Apple!! (Larto)](https://osu.ppy.sh/beatmapsets/14475)
- [Masayoshi Minoshima feat. nomico - Bad Apple!! (ouranhshc)](https://osu.ppy.sh/beatmapsets/18260)
- [Spiritsoulxx - Bad Apple!! (Tony)](https://osu.ppy.sh/beatmapsets/23760)
- [Kommisar - Bad Apple!! (Chiptune Ver.) (Sushi)](https://osu.ppy.sh/beatmapsets/28222)

> - [Kalafina - Bad MagiApple (Makar8000)](https://osu.ppy.sh/beatmapsets/32003)

Reviewer 3: There is this one too https://osu.ppy.sh/beatmapsets/299000#osu/670892

Author: afaict that map wasn't listed in the official rankings

Reviewer 3: What do you mean by "official ranking"?

Reviewer: Those mapsets were listed in the [Beatmap Charts](https://osu.ppy.sh/rankings/osu/charts?spotlight=50) of that time. Unfortunately, Neil's Bad Apple couldn't make it there.

Reviewer 3: Ah okay my bad I thought it was like all the maps that were ranked on that period

The "Bad Apple!!" music video had been some sort of an in-joke at the time with remixes of the song combined with other ironic twists being a prevalent occurrence in beatmaps at the time, hence its appearance as an April Fools' joke. <!--citation needed-->

---

## 2013

### BanchoBot becomes a tsundere

> On 1 April 2013, [BanchoBot](/wiki/BanchoBot) was turned into a [tsundere](https://en.wikipedia.org/wiki/Tsundere "Wikipedia"). On this April Fools' day, any time a user would issue a command to BanchoBot or prompt it to speak in a public chat, its chat messages would be replaced with stereotypical tsundere-like responses to a romantic interest [[24,25,26,27,28,29]][r].

Reviewer 2: ```suggestion
On 1 April 2013, [BanchoBot](/wiki/BanchoBot) was turned into a [tsundere](https://en.wikipedia.org/wiki/Tsundere "Wikipedia"). On this April Fools' day, every time a user would issue a command to BanchoBot or prompt it to speak in a public chat, its chat messages would be replaced with stereotypical tsundere-like responses to a romantic interest [[24,25,26,27,28,29]][r].
```

![](img/2013-regular-banchobot.jpg "Interactions with BanchoBot in 2019")

![](img/2013-tsundere-banchobot.jpg "Interactions with BanchoBot on 1 April 2013")

---

## 2014

### Shiba Inu appears in osu!

On 1 April 2014, the osu! main menu screen was temporarily changed (shown below) to include multi-coloured, grammatically incorrect statements accompanied by the famous image of a certain [Shiba Inu](https://en.wikipedia.org/wiki/Shiba_Inu "Wikipedia") in the style of the [Doge meme](https://en.wikipedia.org/wiki/Doge_(meme) "Wikipedia") that was popular at that time [[14,30,31,32]][r].

![](img/2014-main-screen-doge.jpg "The main screen of osu! on 1 April 2014")

---

## 2015

### osu!coins

*See also: [osu!coin](/wiki/osu!coin)*

> ![](img/2015-osucoins.png "Three osu! coins")

Reviewer 2: ```suggestion
![](img/2015-osucoins.png "The osu! coins")
```

> At the current rate of return, it would be unlikely for me to purchase a personal jet within my lifetime, which is one of my primary life goals. Thus I held discussions with the team on alternate forms of monetisation, studying the current trends in similar free-to-play games currently available on the market.

— peppy, "osu!coins" [[33]][r]

*Note: Under [Coordinated Universal Time (UTC)](https://en.wikipedia.org/wiki/Coordinated_Universal_Time "Wikipedia"), the post was posted on 31 March 2015. However, at the time of the publication, peppy was currently living in Australia, where the current date was 1 April 2015.*

In short, in order to play or restart a [beatmap](/wiki/Beatmap), users must have spent one osu!coin, and once they ran out of osu!coins, they would either have to stop playing and wait until the next day, or pay real money to get more osu!coins. In spite of this description, however, actual gameplay was not affected, and users could just continue playing as normal, even if all their osu!coins were depleted [[33,35,36,37]][r].

> The main screen of osu! also had a barrage of slowly rising osu!coins in the background, along with a slightly changed main theme, in which the exclamation "circles!" was replaced with a robotic-sounding "and buy the coins" before the beat drop. Additional new textures, sound effects, animations, and music were created for this prank, including a counter that would display the amount of coins players had throughout their plays. [[37,38]][r]. <!--needs further verification; also not sure if this paragraph is necessary or not-->

Reviewer 2: ```suggestion
The main screen of osu! also had a barrage of slowly rising osu!coins in the background, along with a slightly changed main theme, in which the exclamation "circles!" was replaced with a robotic-sounding "and buy the coins" before the beat drop. Additional new textures, sound effects, animations, and music were created for this prank, including a counter that would display the amount of coins players had throughout their plays. [[37,38]][r]. <!--needs further verification-->
```

---
tags:
  - april fools
  - april 1st
  - 1 april
  - april 1
  - april fools day
  - joke
  - history
  - legacy
---

# List of osu! April Fools' day jokes

*For the complete history of osu!, see: [History of osu!](/wiki/History_of_osu!)*

> Every year, the osu! developers like to play pranks on the community on April Fools' day. This article contains a list of all the April Fools' day jokes that were played on the osu! community since 2009.

Reviewer: ```suggestion
Every year, the [osu!team](/wiki/People/The_Team) likes to play pranks on the community on April Fools' day. This article contains a list of all the April Fools' day jokes that were played on the osu! community since 2009.
```

## 2009

### "Lemon Tree" gets ranked

> The [beatmap](/wiki/Beatmap) "Best of No.1 Hits - Lemon Tree (MillhioreF)" was [ranked](/wiki/Beatmap/Category#ranked) on 1 April 2009 as part of that year's April Fools' joke. The beatmap was very much an in-joke within the community with users sarcastically claiming that such a beatmap should be kept ranked forever \[1,2,3]\. It's ranked status was removed by moderators shortly afterwards \[2\].

Author: assuming you meant 
```suggestion
The [beatmap](/wiki/Beatmap) ["Best of No.1 Hits - Lemon Tree (MillhioreF)"](https://osu.ppy.sh/beatmapsets/57878#osu/174267) was [ranked](/wiki/Beatmap/Category#ranked) on 1 April 2009, as part of that year's April Fools' joke. The beatmap was very much an in-joke within the community with users sarcastically claiming that such a beatmap should be kept ranked forever \[1,2,3\]. Its ranked status was removed by the moderation team shortly afterwards \[2\].
```

After some time, the original beatmap was removed on request of its creator; however, it was eventually reuploaded on 24 Aug 2012 by [MillhioreF](https://osu.ppy.sh/users/941094) for archival purposes \[4\].

#### References

1. [Reddit post (r/osugame) [u/5522Luca] - "Reminder the Osu! April Fools 2009? This beatmap was ranked."](https://www.reddit.com/r/osugame/comments/64it62/reminder_the_osu_april_fools_2009_this_beatmap/ "Reddit")
2. [osu! forum comment (Beatmap Graveyard) [machol30] - "Best of No.1 Hits - Lemon Tree"](https://osu.ppy.sh/community/forums/topics/95725?start=106774)
3. [osu! forum comment (Beatmap Graveyard) [peppy] - "Best of No.1 Hits - Lemon Tree"](https://osu.ppy.sh/community/forums/topics/95725?start=105679)
4. [osu! beatmap - "Best of No.1 Hits - Lemon Tree" (MillhioreF)](https://osu.ppy.sh/beatmapsets/57878#osu/174267)

---

## 2010

### touhosu!

> For the 2010 April Fools' prank, the osu! website along with the in-game main menu screen was changed to theme with characters and references from the [Touhou Project](https://en.wikipedia.org/wiki/Touhou_Project "Wikipedia") (shown below). The changes included the addition of the character Marisa Kirisame and a display of differently-coloured butterflies in a circular pattern across the main menu and website, along with the swapping of the name "osu!" to "touhosu!" in certain areas across the website (shown below) \[1,2,3\].

Author: this was intentional; using "theme" as a verb here

> For the 2010 April Fools' prank, the osu! website along with the in-game main menu screen was changed to theme with characters and references from the [Touhou Project](https://en.wikipedia.org/wiki/Touhou_Project "Wikipedia") (shown below). The changes included the addition of the character Marisa Kirisame and a display of differently-coloured butterflies in a circular pattern across the main menu and website, along with the swapping of the name "osu!" to "touhosu!" in certain areas across the website (shown below) \[1,2,3\].

Reviewer: ah, didn't notice. both forms seem to be applicable here though.

2. [Discord message (osu!dev) [Nivalyx#9577] - 27 January 2021 (05:06 UTC)](https://discord.com/channels/188630481301012481/218677502141399041/804215894762848296 "Discord")
3. [osu! forum thread (Resolved Issues) [rcmero] - "touhousu! - April Fools joke? [Resolved]"](https://osu.ppy.sh/community/forums/topics/27612)
4. [osu! wiki article - "Touhosu!"](/wiki/Glossary/Touhosu!)
5. [osu! forum thread (General Discussion) [rulingvenus] - "Naked Marisa????"](https://osu.ppy.sh/community/forums/topics/27531)

---

## 2011

### osu!core

"osu!core" is the name given to osu!'s 2011 April Fools' day prank. The prank caused every single [beatmap](/wiki/Beatmap)'s audio to be pitched and sped up in the style of [Nightcore](https://en.wikipedia.org/wiki/Nightcore "Wikipedia") remixes. Although just an April Fools' day prank, it later became a reality when the [Nightcore mod](/wiki/Game_modifier/Nightcore) was introduced as an actual playable [game modifier](/wiki/Game_modifier) in osu! later on \[1,2,3,4\].

> #### References

Reviewer 4: Instead of creating seperate references here, use one big references section at the bottom. We want everyone to focus on the article content firsthand.

- [Masayoshi Minoshima ft. nomico - Bad Apple!! (ignorethis)](https://osu.ppy.sh/beatmapsets/13177)
- [Masayoshi Minoshima feat. StrawbellyCake - Bad Apple!! (German Version) (Larto)](https://osu.ppy.sh/beatmapsets/13664)
- [Masayoshi Minoshima feat. Larto & nomico - Awesome Apple!! (Larto)](https://osu.ppy.sh/beatmapsets/14475)
- [Masayoshi Minoshima feat. nomico - Bad Apple!! (ouranhshc)](https://osu.ppy.sh/beatmapsets/18260)
- [Spiritsoulxx - Bad Apple!! (Tony)](https://osu.ppy.sh/beatmapsets/23760)
- [Kommisar - Bad Apple!! (Chiptune Ver.) (Sushi)](https://osu.ppy.sh/beatmapsets/28222)
- [Kalafina - Bad MagiApple (Makar8000)](https://osu.ppy.sh/beatmapsets/32003)

The "Bad Apple!!" music video had been some sort of an in-joke at the time with remixes of the song combined with other ironic twists being a prevalent occurrence in beatmaps at the time, hence its appearance as an April Fools' joke. <!--citation needed-->

#### References

1. [Bad Apple!! Ranking Chart, 4 April 2012](https://web.archive.org/web/20120404051938/http://osu.ppy.sh/p/chart?ch=BADAPPLE "Internet Archive — osu!")

> 2. [osu! newspost [Cyclone] (1 April 2012) - "Bad Apple!! Ranking Chart"](https://osu.ppy.sh/community/forums/topics/79128?start=1431905)

Author: both of these are admin forbidden, but if im not mistaken isn't 1431905 the number attributed to start and not the post number?

> 2. [osu! newspost [Cyclone] (1 April 2012) - "Bad Apple!! Ranking Chart"](https://osu.ppy.sh/community/forums/topics/79128?start=1431905)

Reviewer: that is the permalink that goes to the forum post specifically, rather than the topic. both redirect to the same page and same post, but my suggestion contains a clearer one

![](img/2013-regular-banchobot.jpg "Screenshot of a player interaction with BanchoBot in 2019")

![](img/2013-tsundere-banchobot.jpg "Screenshot of a player interaction with BanchoBot on 1 April 2013")

#### References

1. [Reddit comment (r/osugame) [u/Saluya_Lv9] - "April 1st"](https://www.reddit.com/r/osugame/comments/2201so/april_1st/ "Reddit")
2. [osu! forum comment (Off-Topic) [Jazz] - "Your prediction of osu! April Fools"](https://osu.ppy.sh/community/forums/topics/125391?start=2215004)
3. [osu! forum comment (Off-Topic) [Brian OA] - "Your prediction of osu! April Fools"](https://osu.ppy.sh/community/forums/topics/125391?start=2215194)
4. [Tweet [@little_2d] - 27 June 2019](https://twitter.com/little_2d/status/1144316731407683584 "Twitter")
5. [osu! forum comment (Development) [kingking9] - "osu! Community Localisation Project"](https://osu.ppy.sh/community/forums/topics/104342?start=2342998)

> 6. [osu! forum comment (Development) [peppy] - "osu! Community Localisation Project"](https://osu.ppy.sh/community/forums/topics/104342?start=2343044)

Author: I mainly left these citations in as proof of hearsay. It's very unlikely that a group of seemingly unrelated accounts across different forums would come together to spread some malicious false info about some random april fools' joke from 2013.

Plus, the only citation that is "definitive proof" that this joke did exist was the peppy comment, and even that could be questionable as the comment didn't mention april fools' directly and is only assuming that "ts" is short for tsundere

![](img/2014-main-screen-doge.jpg "Screenshot of the osu! main screen on 1 April 2014")

#### References

1. [Reddit comment (r/osugame) [u/mystry08] - "Can we save the start screen doge?"](https://www.reddit.com/r/osugame/comments/21vh6r/can_we_save_the_start_screen_doge/ "Reddit")
2. [Reddit comment (r/osugame) [u/dalollypop] - "Very April, Such fool, Much peppy. wow"](https://www.reddit.com/r/osugame/comments/21u293/very_april_such_fool_much_peppy_wow/ "Reddit")
3. [osu! forum thread (Off-Topic) [Decuke] - "Doge on Osu!"](https://osu.ppy.sh/community/forums/topics/198112)
4. [Discord message (osu!dev) [Тик#5580] - 28 January 2021 (08:05 UTC)](https://discord.com/channels/188630481301012481/218677502141399041/804260860331687946 "Discord")

---

## 2015

### osu!coins

*See also: [osu!coin](/wiki/osu!coin)*

![](img/2015-osucoins.png "Three osu! coins")

> On 31 March 2015, [peppy](https://osu.ppy.sh/users/2) released [a newspost](https://osu.ppy.sh/home/news/2015-03-31-osucoins) that announced the addition of a new in-game currency to be added to osu!, dubbed "osu!coins" \[1,2\]. The newspost explained what the in-game currency was and how it worked along with a specially-created [osu!academy video](https://www.youtube.com/watch?v=BImc5McuK1o "YouTube"). Along with this, peppy also joked that his reasoning behind the change was due to the fact that the current monetary gain from player donations would not have been enough to buy him a personal jet within his lifetime:

Author: oh god have i been spelling it "newspost" this entire time? where did i get that?

ill add a fix for this in a separate commit with citation fixes

> At the current rate of return, it would be unlikely for me to purchase a personal jet within my lifetime, which is one of my primary life goals. Thus I held discussions with the team on alternate forms of monetisation, studying the current trends in similar free-to-play games currently available on the market.

— peppy, "osu!coins" \[1\]

*Note: Under [Coordinated Universal Time (UTC)](https://en.wikipedia.org/wiki/Coordinated_Universal_Time "Wikipedia"), the post was posted on 31 March 2015. However, at the time of the publication, peppy was currently living in Australia, in which the current date was 1 April 2015.*

In short, in order to play or restart a [beatmap](/wiki/Beatmap), users must spend one osu!coin, and once they ran out of osu!coins, they would either have to stop playing and wait until the next day, or pay real money to get more osu!coins. In spite of this description, however, actual gameplay was not affected, and users could just continue playing as normal once all their osu!coins were depleted \[1,3,4,5\].

> A new osu! main screen was also made that had a barrage of slowly rising osu!coins in the background, along with a slightly changed main theme, in which the exclamation "circles!" was replaced with a robotic-sounding "and buy the coins" before the beat drop. Additional new textures, sound effects, animations, and music were created for this prank, including a counter that would display the amount of coins players had throughout their plays. \[5,6\]. <!--needs further verification; also not sure if this paragraph is necessary or not-->

Reviewer 2: ```suggestion
The main screen of osu! also had a barrage of slowly rising osu!coins in the background, along with a slightly changed main theme, in which the exclamation "circles!" was replaced with a robotic-sounding "and buy the coins" before the beat drop. Additional new textures, sound effects, animations, and music were created for this prank, including a counter that would display the amount of coins players had throughout their plays. \[5,6\]. <!--needs further verification; also not sure if this paragraph is necessary or not-->
```

#### References

1. [osu! newspost [peppy] (31 March 2015) - "osu!coins!"](https://osu.ppy.sh/home/news/2015-03-31-osucoins)
2. [April Fools' Day On The Web entry - "osu!coins! (fake business model, obviously a joke from blog & video)"](http://aprilfoolsdayontheweb.com/joke/20150013/?size=1 "April Fools' Day On The Web")
3. [osu! wiki article - "osu!coin"](/wiki/osu!coin)
4. [YouTube video [synonia] - "Osu! Coin generator 14 coins in 30 seconds"](https://www.youtube.com/watch?v=Cmt646ujDFc "YouTube")
5. [YouTube video [osu!] - "Introduction to osu!coins (April Fools'2015)"](https://www.youtube.com/watch?v=BImc5McuK1o "YouTube")
6. [YouTube video [BananCho] - "Osu!Coins."](https://www.youtube.com/watch?v=0yWlUzG_tb8&t=39s "YouTube")
7. [YouTube video [TheRexster] - "HOW TO GET OSU COINS VERY FAST!"](https://www.youtube.com/watch?v=wRVd5Bdf9rk "YouTube")
8. [osu! forum thread (General Discussion) [Terriama] - "April Fools"](https://osu.ppy.sh/community/forums/topics/377157)
9. [osu! changelog comment [peppy] - "Cutting Edge 20150401"](https://osu.ppy.sh/comments/121803)

---

## 2016

### osu! in virtual reality

> On 1 April 2016, [a newspost](https://osu.ppy.sh/home/news/2016-04-01-oculus-rift-to-be-supported-as-an-input-method) was published detailing the announcement of plans to add support for the [Oculus Rift](https://en.wikipedia.org/wiki/Oculus_Rift "Wikipedia") as a new [input method](/wiki/Gameplay/Input_device) in osu!. The post, written by [Evrien](https://osu.ppy.sh/users/791660), referenced many quotes from a supposed interview of [peppy](https://osu.ppy.sh/users/2) in which he explained his reasoning for the announcement and ideas for how the concept might work \[1\].

Reviewer 2: ```suggestion
On 1 April 2016, [a news post](https://osu.ppy.sh/home/news/2016-04-01-oculus-rift-to-be-supported-as-an-input-method) was published detailing the announcement of plans to add support for the [Oculus Rift](https://en.wikipedia.org/wiki/Oculus_Rift "Wikipedia") as a new [input method](/wiki/Gameplay/Input_device) in osu!. The post, written by [Evrien](https://osu.ppy.sh/users/791660), referenced many quotes from a supposed interview of [peppy](https://osu.ppy.sh/users/2) in which he explained his reasoning for the announcement and ideas for how the concept might work \[1\].
```

> Regarding how players might use the Oculus Rift as an input method, the newspost describes that "the player will experience a first person point-of-view of the cursor as it travels to and from hit-objects on screen…" and will hit hit objects by "…having the player produce vowel-like sounds using their mouth". No actual in-game changes were made relating to using the Oculus Rift or a similar [virtual reality (VR)](https://en.wikipedia.org/wiki/Virtual_reality "Wikipedia") device to control osu! \[1\].

Reviewer 2: ```suggestion
Regarding how players might use the Oculus Rift as an input method, the news post describes that "the player will experience a first person point-of-view of the cursor as it travels to and from hit-objects on screen…" and will hit hit objects by "…having the player produce vowel-like sounds using their mouth". No actual in-game changes were made related to using the Oculus Rift or a similar [virtual reality (VR)](https://en.wikipedia.org/wiki/Virtual_reality "Wikipedia") device to control osu! \[1\].
```

*Notice: McOsu is developed separately from and has no direct affiliation with osu! or ppy Pty Ltd.*

> However, even though the official osu! developers have/had no intention of genuinely adding VR support, the idea of osu! in VR piqued interest in the minds of some fans. This interest eventually culminated into an unofficial fan project that was started shortly afterwards that aimed to create a free and open source client to practice osu! [beatmaps](/wiki/Beatmap) in with more features and [game modifiers](/wiki/Game_modifier) including the ability to play in VR. Named "[McOsu](https://store.steampowered.com/app/607260/McOsu "Steam")", the project was completed and released on [Steam](https://en.wikipedia.org/wiki/Steam_(service) "Wikipedia") on 20 March 2017 \[2,3,4,5\].

Author: i had searched around and it seemed that "open source" was the more commonly favoured spelling

#### References

1. [osu! newspost [Evrien] (1 April 2016) - "Oculus Rift to be Supported as an Input Method (April Fools!)"](https://osu.ppy.sh/home/news/2016-04-01-oculus-rift-to-be-supported-as-an-input-method)
2. [Reddit post (r/osugame) [u/Omgforz] - "McOsu Alpha 20 Public release (custom practice client)"](https://www.reddit.com/r/osugame/comments/4vuksd/mcosu_alpha_20_public_release_custom_practice/ "Reddit")
3. [YouTube video [Omgforz] - "McOsu Alpha 20 (custom practice client +download)"](https://www.youtube.com/watch?v=PCLpOdcMQuc "YouTube")
4. [GameSkinny - "What Even Is McOsu? Because It's Not Osu!"](https://www.gameskinny.com/mhaa0/what-even-is-mcosu-because-its-not-osu "GameSkinny")
5. [GitHub - "McKay42/McOsu"](https://github.com/McKay42/McOsu "GitHub")

### Dancing Auto mod cursor/dancing pippi

> "dancing pippi" (also known as "dancing Auto mod cursor") is the nickname given to one of osu!'s April Fools' jokes for 2016 in which an update was released that caused the game cursor in [Auto](/wiki/Game_modifier/Auto) mod [replays](/wiki/Replay) to circle around the current [hit object](/wiki/Hit_object) in pixel-perfect fashion before eventually hitting the object right on time, which is in contrast to the Auto mod's usual robotic and perfectly straight movements. The aforementioned update was rolled back in a subsequent one the next day \[1,2,3,4\]. <!--still needs official official verification-->

Reviewer 2: ```suggestion
"dancing pippi" (also known as "dancing Auto mod cursor") is the nickname given to one of osu!'s April Fools' jokes for 2016 in which an update was released that caused the game cursor in [Auto](/wiki/Game_modifier/Auto) mod [replays](/wiki/Replay) to circle around the current [hit object](/wiki/Hit_object) in pixel-perfect fashion before eventually hitting the object right on time, in contrast to the Auto mod's usual robotic and perfectly straight movements. The aforementioned update was rolled back in a subsequent one the next day \[1,2,3,4\]. <!--still needs official official verification-->
```

#### References

1. [YouTube video [HoLLy] - "osu!'s april fools 2016 (auto mod improvement)"](https://www.youtube.com/watch?v=r9SCbYG4GYs "YouTube")
2. [YouTube video [Hubz] - "osu! 2016 april fools (dancing pippi)"](https://www.youtube.com/watch?v=fYTdPqhAns0 "YouTube")
3. [YouTube video [mightyaleks] - "Osu! Dancing Auto-cursor and retard Spin | 1st April 2016"](https://www.youtube.com/watch?v=5Tj-1sgHl9g "YouTube")
4. [Reddit post (r/osugame) [u/osuisgameforweebs] - "Something about the april fools joke dancing that some might not have noticed"](https://www.reddit.com/r/osugame/comments/4crlw1/something_about_the_april_fools_joke_dancing_that/ "Reddit")

### Free osu! supporter tags

> On 1 April 2016 many osu! players were surprised to discover that they had suddenly and inexplicably received an [osu! supporter tag](https://osu.ppy.sh/home/support) despite never having bought or being gifted one. The supporter tag that was given to players was fully functional and acted just like a regular supporter tag, however the change was rolled back the next day \[1,2,3,4,5,6,7\].

Reviewer: ```suggestion
On 1 April 2016 many osu! players were surprised to discover that they had suddenly and inexplicably received an [osu! supporter tag](https://osu.ppy.sh/home/support) despite never having bought or being gifted one. The supporter tag that was given to players was fully functional and acted just like a regular supporter tag; however, the change was rolled back the next day \[1,2,3,4,5,6,7\].
```

#### References

1. [Reddit post (r/osugame) [u/[deleted]] - "New April Fools Update now has a rotating osu! Logo"](https://www.reddit.com/r/osugame/comments/88kq23/new_april_fools_update_now_has_a_rotating_osu_logo/?st=JFGBCI5G&sh=49cf8677 "Reddit")
2. [Reddit post (r/osugame) [u/hi_im_marc] - "April Fools Patch Is Out Get Ready To Get BAMBOOZLED!!!1"](https://www.reddit.com/r/osugame/comments/88kbit/april_fools_patch_is_out_get_ready_to_get/ "Reddit")
3. [Reddit post (r/osugame) [u/AdriaLOL] - "haha, nice april fools peppy XD"](https://www.reddit.com/r/osugame/comments/88qlwk/haha_nice_april_fools_peppy_xd/ "Reddit")
4. [osu! forum thread (General Discussion) [Aochie] - "The osu! logo is moving?"](https://osu.ppy.sh/community/forums/topics/724377)
5. [osu! forum thread (Resolved Issues [Jreen] - "[resolved] Osu! Logo Sideways?"](https://osu.ppy.sh/community/forums/topics/724094)
6. [osu! wiki article - "History of osu! 2018"](/wiki/History_of_osu!/2018#april)

---

## 2019

### Sneezing girl sound effect (2019)

> For April Fools' day 2019 there was approximately a 1 in 20 chance of hearing the sound effect of a high-pitched girl sneezing upon opening a [beatmap](/wiki/Beatmap) \[1,2,3\].

Reviewer: ```suggestion
For April Fools' day 2019 there was approximately a 1 in 20 chance of hearing the sound effect of a high-pitched girl sneezing upon opening a beatmap \[1,2,3\].
```

#### References

1. [Reddit post (r/osugame) [u/jivko500] - "The April Fools joke in osu"](https://www.reddit.com/r/osugame/comments/b83pnl/the_april_fools_joke_in_osu/ "Reddit")
2. [Reddit post (r/osugame) [u/anoymaly2152] - "Bless you, Pippi."](https://www.reddit.com/r/osugame/comments/b848ro/bless_you_pippi/ "Reddit")
3. [osu! forum thread (Off-Topic) [Brainage] - "No April Fools in the changelog?"](https://osu.ppy.sh/community/forums/topics/888939)

---

## 2020

> ### MillhioreF joins the Featured Artists

Reviewer: There needs to be noted somewhere down that [Waltz o' the Irish](https://osu.ppy.sh/beatmapsets/73348) was moved to the Loved section that day as part of the April Fools' joke.

Author: I'll look through my sources, but do you have any direct proof of this?

Also assuming that it was moved back out of Loved the same day?

Reviewer: I didn't bother providing you with any source to this, since it's already noted on the newspost. It is still in Loved section, wasn't moved back out.

Author: Oh, totally my bad lol. This just snuck past me ig

> [MillhioreF](https://osu.ppy.sh/users/941094)—a long-time osu! moderator, developer and [Easy mod](/wiki/Game_modifier/Easy) player—was announced in [a newspost](https://osu.ppy.sh/home/news/2020-04-01-new-featured-artist-millhioref) to have "joined" the list of [Featured Artists](/wiki/Featured_Artists) as "Millhiore Firianno Biscotti" on 1 April 2020 with a selection of five songs to boot \[1\]:

Reviewer: You can also add a detail about millhi's featured artist ID, which was 727, though the page no longer exists.

![](https://zeu.s-ul.eu/awYdE8En.png)

Author: Ehhh. I typically enjoy added bits of trivia but this is just a bit too trivial imo. If anyone is really in dire need of this info they'll know how to find it
